examples/typescript/validateBranch.ts

```diff
@@ -0,0 +1,57 @@
+import {LookerNodeSDK} from '@looker/sdk-node'
+import {IProjectError} from '@looker/sdk'
+
+
+const sdk = LookerNodeSDK.init40()
+
+
+// this will validate lookml in development mode
+const validateBranch = async (projectName: string, branch: string): Promise<IProjectError[]> => {
+
+    // check current workspace, if prod switch to dev (needed to be able to checkout a different git branch)
+    const workspace = await sdk.ok(sdk.session());
+    console.log(JSON.stringify(workspace))
+
+    if (workspace.workspace_id == 'production') {
```

Comment:
So workspace_id will always be production or dev?

If so, api documentation in helltool should be updated to reflect this! Unrelated to your example but your example did prompt me to look in API explorer.